Makefile.toml

```diff
@@ -9,8 +9,8 @@ skip_core_tasks = true
 RUST_OPTIMIZER_VERSION = "0.15.0"
 # Use rust version from rust-optimizer Dockerfile (see https://github.com/CosmWasm/rust-optimizer/blob/main/Dockerfile#L1)
 # to be sure that we compile / test against the same version
-RUST_VERSION = "1.69.0"
-NIGHTLY_VERSION = "nightly-2023-08-29"
+RUST_VERSION = "1.75.0"
```

Comment:
Should maybe use 1.72.0 like the comment above says.

Reply:
updated rust optimizer version to 0.15.1 which uses 1.75.0

Reply:
yeah but we use `abstractmoney/rust-optimizer` which has no 0.15.0.